src/users/users.spec.ts

```diff
@@ -193,15 +193,18 @@ describe('UserManagement', () => {
       });
     });
 
-    describe('completeEmailVerification', () => {
+    describe('verifyEmail', () => {
       it('sends a Complete Email Verification request', async () => {
-        mock.onPost(`/users/email_verification`).reply(200, userFixture);
+        mock.onPost(`/users/user-123/verify_email`).reply(200, userFixture);
 
-        const resp = await workos.users.completeEmailVerification(
-          'email-verification-token',
-        );
+        const resp = await workos.users.verifyEmail({
+          userId: 'user-123',
```

Comment:
nit: Can we make this match the structure of the IDs we'll see in the wild?

```suggestion
          userId: 'user_123',
```